### PR TITLE
update: checkpoint

### PR DESCRIPTION
**Pull request by decryp2kanon**
mainnet at `6513497` 2020-09-04
- :heavy_check_mark: https://1explorer.sugarchain.org/block/855f0c66238bc0246c8ca25cf958283fd49b9fb4b217ddeb518e5ea9f5071b9e

testnet at `4000000`
- :x:  https://1explorer-testnet.cryptozeny.com/block/bc05c2d5e81785f287cd58a798b64467cff35c8ef2bbe8062d8420eeb86f4056 
- :heavy_check_mark: https://api-testnet.sugarchain.org/height/4000000

### CODE REVIEW

**Review by y-chan — APPROVED** [review]
LGTM

File: src/chainparams.cpp [hunk 1]
```diff
@@ -215,14 +215,13 @@ class CMainParams : public CChainParams {
 
         // The best chain should have at least this much work.
         // TODO.SUGAR.UPDATE
-        // (510*510*17)+1=4421701
-        // getblockhash 4421701 && "chainwork"
-        consensus.nMinimumChainWork = uint256S("0000000000000000000000000000000000000000000000000000214f0c90bb88");
+        // getblockhash 6513497 && "chainwork"
+        consensus.nMinimumChainWork = uint256S("00000000000000000000000000000000000000000000000000003f23ef34da28");
```
Comment (Nugetzrul3) [inline src/chainparams.cpp:219]: This is correct 👍

File: src/chainparams.cpp [hunk 2]
```diff
@@ -284,16 +283,17 @@ class CMainParams : public CChainParams {
                 {900000, uint256S("8a566a463925cf028cb427edb6d4d18a9c1213bf472d9032369c4e45449eb71c")},
                 {2601511, uint256S("e30af0fcf522354f43c3ab2cfb4805d175b1264608b6124eab0b635bdea2589d")}, // 2601511=2601001+510
                 {4422211, uint256S("1f4186606aaab3cb3818f073599602e573476da9fed0b65c9f14646210d6b18f")}, // 4422211=4421701+510
+                {6513497, uint256S("855f0c66238bc0246c8ca25cf958283fd49b9fb4b217ddeb518e5ea9f5071b9e")},
```
Comment (Nugetzrul3) [inline src/chainparams.cpp:286]: This is correct

**Review by Nugetzrul3 — APPROVED** [review]
LGTM

**Pull request merged**